tests/model/team_test.py

```diff
@@ -64,3 +64,9 @@ def test_is_member():
     assert team.is_member(new_slack_id) is False
     team.add_member(new_slack_id)
     assert team.is_member(new_slack_id)
+
+
+def test_print():
+    """Test print team class."""
+    team = Team("brussel-sprouts", "Brussel Sprouts")
+    print(team)
```

Comment:
A better test here might be to get the string from the Team and checking if its what you expect, rather than printing it (where you would have to check by hand that the unit test succeeded).

Reply:
How do you get the team string without printing it?

Reply:
Call `__str__()` directly or look into the formatting functions in Python.

Reply:
Would like to add that calling `str(team)` also works.